src/App/App.js

```diff
@@ -0,0 +1,22 @@
+import { useContext, useEffect } from "react";
+import { Routes, Route } from "react-router-dom";
+import { Context } from "src/index";
+import Registration from "src/components//Registration/Registration";
+import Login from "src/components/Login/Login";
+import 'src/App/app.scss'
+
+const App = () => {
+  const { store } = useContext(Context);
+
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Login />} />
+        <Route path="/registration" element={<Registration />} />
+        <Route path="/login" element={<Login />} />
```

Comment:
Why did you install one component for two paths? Use a redirect for the first path (or the Navigate component for v6 )